docs/README.md

```diff
@@ -0,0 +1,290 @@
+
+**What if you only have the name or symbol of the project?**
+
+That's where subgraph queries come in handy. (and we have plenty of those 😉) - But feel free to also [create your own](#custom-queries).
+
+**_Make sure all addresses are input in lowercase letters, as the queries are case sensitive._**
+
+## Fetching a TCO2 by its symbol
+
+```typescript
+const tco2 = await toucan.fetchTCO2TokenByFullSymbol("TCO2-VCS-1718-2013");
+```
+
+The result will look like so:
+
+```json
+{
+  "id": "0x0044c5a5a6f626b673224a3c0d71e851ad3d5153",
+  "name": "Toucan Protocol: TCO2-VCS-1718-2013",
+  "symbol": "TCO2-VCS-1718-2013",
+  "address": "0x0044c5a5a6f626b673224a3c0d71e851ad3d5153",
+  "projectVintage": {
+    "name": "2013",
+    "project": {
+      "projectId": "VCS-1718"
+    }
+  }
+}
+```
+
+Now you have quite some info on the project, including its address.
+
+## All queries:
+
+Toucan SDK offers a lot of pre-defined queries. Try them out!
+
+```typescript
+// Fetch the batches of a user.
+fetchUserBatches(address: string);
+
+// Fetch TCO2 token by it's address to get details like name, symbol, address and projectVintage.
+fetchTCO2TokenById(address: string);
+
+// Fetch TCO2Token with the symbol to get details like name, symbol, address and projectVintage. A symbol would look like this: `TCO2-VCS-<projectId>-YYYY`. Make sure to input the full symbol.
+fetchTCO2TokenByFullSymbol(symbol: string);
+
+// Fetch all TCO2s details like name, symbol, address and projectVintage.
+
+fetchAllTCO2Tokens();
+
+// Fetch all user retirements to get information like creationTX, amount, token details and certificate details.
+fetchUserRetirements(address: string, first?: number, skip?: number);
+
+// Fetch all redeems of a given pool to get information like amount, creator and token details. Only accepts the Pooltypes "NCT" or "BCT".
+fetchRedeems(poolSymbol: Pooltypes, first?: number, skip?: number);
+
+// Fetch all redeems of a user by pool and get information like amount and token details.
+fetchUserRedeems(address: string, poolSymbol: Pooltypes, first?: number, skip?: number);
+
+// Fetch TCO2 tokens that are part of the given pool and get information on name, amount, methodology and more.
+fetchPoolContents(poolSymbol: Pooltypes, first?: number, skip?: number);
+
+// Fetch a project by it's id to get more info on region and the standard.
+
+fetchProjectById();
+
+// Fetch all aggregations (including for example, tco2TotalRetired or totalCarbonBridged).
+fetchAggregations();
+```
+
+## Custom queries
+
+There's a lot more other pre-built subgraph queries that I could show you, but what I really want to show you is the `fetchCustomQuery` method.
+
+This allows you to fetch with your own queries and can be very powerful if you know graphQL. You can also check out a lot of example queries in our subgraph [playgrounds](https://thegraph.com/hosted-service/subgraph/toucanprotocol/matic).
+
+- Getting all infos on a project of a Carbon Credit
```

Comment:
"[...] on a carbon project"

Reply:
Might also make sense to tell people that `region` is not actually the region but the country :smiling_face_with_tear: